### PR TITLE
Can save boot content in loadout

### PR DESCRIPTION
**Pull request by BraveMole**
## About The Pull Request

Closes: https://github.com/tgstation/TerraGov-Marine-Corps/issues/7407

## Changelog
:cl:
expansion:  Can save boot content in loadout
/:cl:

### CODE REVIEW

**Comment by MereWalker** [comment]
Could have used that in the "walking dead" event
Thanks!

File: code/datums/loadout/item_representation/_item_representation.dm [hunk 1]
```diff
@@ -148,3 +148,30 @@
...
+/datum/item_representation/boot
+	/// The item stored in the boot
+	var/datum/item_representation/boot_content
+
+/datum/item_representation/boot/New(obj/item/item_to_copy)
+	if(!item_to_copy)
+		return
+	if(!istype(item_to_copy, /obj/item/clothing/shoes/marine))
+		CRASH("/datum/item_representation/boot created from an item that is not a marine boot")
+	..()
```
Comment (hyper2snyper) [inline code/datums/loadout/item_representation/_item_representation.dm:161]: `. = ..()` ?

Comment (hyper2snyper) [inline code/datums/loadout/item_representation/_item_representation.dm:161]: `. = ..()` ?

**Review by hyper2snyper — APPROVED** [review]
Ultimately I figure we should make a global representation of items with storage so we do not have to do this every time. This works for now though.

File: code/datums/loadout/item_representation/_item_representation.dm (hunk 1 above)
Comment (BraveMole) [reply code/datums/loadout/item_representation/_item_representation.dm:161]: nop, no in new

**Review by TiviPlus — APPROVED** [review]
There's a snake in my boot

**Pull request merged**